README.md

```diff
@@ -129,6 +129,20 @@ On the server side, the params will be received as:
 `"filters"=>{"id"=>"[1,2]"}` compared to the standard format of
 `"filters"=>{"id"=>["1", "2"]}`.
 
+Note that this feature cannot currently be wrapped in an array and should not be used in combination with sending array parameters individually.
```

Comment:
This doesn't render as beautifully as it looks in the code. Maybe change snippets to ` ```ruby`